devices/common_patches/g4_adc_common.yaml

```diff
@@ -0,0 +1,11 @@
+ADC*_*:
+  CCR:
+    _add:
+      VBATEN:
+        description: V_BAT battery voltage channel selection
+        bitWidth: 1
+        bitOffset: 24
+      VSENSEEN:
```

Comment:
Should this be `VSENSESEL`? `peripherals/adc/adc_v3_common_g4.yaml` would also need updating if so.